malsim/sims/mal_simulator.py

```diff
@@ -393,6 +394,16 @@ def register_defender(self, agent_name):
         self.possible_agents.insert(0, agent_name)
         self.agents_dict[agent_name] = {"type": "defender"}
 
+    def get_attacker_agents(self):
+        """Return agents_dict but only with attacker agents"""
+        return {k: v for k, v in self.agents_dict.items()
+                if v['type'] == "attacker"}
+
+    def get_defender_agents(self):
+        """Return agents_dict but only with defender agents"""
```

Comment:
Rephrase comment to Return agents dictionaries of attacker agents.

And the one bellow Return agents dictionaries of defender agents.

Reply:
Fixed